travis/build-all-manylinux-wheels.sh

```diff
@@ -3,8 +3,10 @@
 set -x
 set -e
 
+arch="${1}"
```

Comment:
Default to `x86_64`, so typing:

```
$ sh travis/build-all-manylinux-wheels.sh
```

Would produce the wheels for the x86_64 architecture.

Something like this may work:

```
ARCH="${1:-x86_64}"
```